<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainView</class>
 <widget class="QDialog" name="MainView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainView</string>
  </property>
  <widget class="QLabel" name="lblVideo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblYCrCb">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>160</width>
     <height>120</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblFiltered">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>520</y>
     <width>160</width>
     <height>120</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblContour">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>520</y>
     <width>160</width>
     <height>120</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblOther">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>520</y>
     <width>160</width>
     <height>120</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblConvexity">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>520</y>
     <width>160</width>
     <height>120</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStart">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>240</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="txtPanel">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>20</y>
     <width>160</width>
     <height>200</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSetting">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>284</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Setting</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
